### PR TITLE
feat: login and signup page

### PR DESCRIPTION
**Pull request by maitrakhatri**
- static login and signup page without any functionalities

### CODE REVIEW

File: gozo-stores/src/pages/LogIn.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                            <div class="input-group">
+                                <label for="email">Email address</label>
+                                <input type="email" name="email" id="emailid" placeholder="admin@gozo.com" required/>
+                            </div>
+                            <div class="input-group">
+                                <label for="password">Password</label>
+                                <input type="password" name="password" id="password" required/>
+                            </div>
+                            <div class="login-options">
+                                <input type="checkbox" name="TnC" id="remember-cred" />Remember me <br/>
```
Comment (OmKakatkar) [inline gozo-stores/src/pages/LogIn.jsx:26]: Write `className` instead of `class`

File: gozo-stores/src/pages/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                    <form action="" method="post">
+                        <div class="card shadow">
+
+                            <h3 class="title">Sign Up</h3>
+                            <div class="input-group">
+                                <label for="email">Email address</label>
+                                <input type="email" name="email" id="emailid" placeholder="admin@gozo.com" required/>
+                            </div>
+                            <div class="input-group">
+                                <label for="password">Password</label>
```
Comment (OmKakatkar) [inline gozo-stores/src/pages/SignUp.jsx:21]: Use `htmlFor` instead of `for`

File: gozo-stores/src/pages/css/auth.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    padding: 6rem;
+}
+
+.auth .card {
+    padding: 2rem 6rem;
+    width: fit-content;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    background-color: white;
```
Comment (OmKakatkar) [inline gozo-stores/src/pages/css/auth.css:13]: Use css variables for colors. This provides consistency

Comment (ranitasahaa1230) [inline gozo-stores/src/pages/css/auth.css:13]: You can declare all these colors in your root variables

**Review by OmKakatkar — APPROVED** [review]
* Have left some recommended changes please have a look.
* Also, I like that you are using barrel imports for the Pages

**Review by anupkgurung — APPROVED** [review]
clean merge.
clean code

**Review by ranitasahaa1230 — APPROVED** [review]
Good use of semantic tags
Code can be understood easily

**Pull request merged**